.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -12,21 +12,20 @@ body:
       options:
         - bug
         - batch
-        - config.writer
+        - flepiconfig
```

Comment:
Is there a reason for choosing `flepiconfig` over just `config`? I'm not contesting the need for this issue, but I think the name referring to an R package contained in this repository is confusing.

Reply:
Mmm, the old issue label was named `config.writer` (which I thought was the name of the old package...?). So I renamed it to coincide with the current R package. I can change it to `config` and broaden the definition for that tag to be 'Relating to configuration files and framework.', if you prefer.

Reply:
Oh, okay, I missed that bit of context, thanks! That's fine then, but also need issues for the other packages. I don't think a generic config label is bad either since both R/python parse config. Does this mean that the `inference` label refers to the R package and not "inference" as a concept then?

Reply:
yeah, we need a `config` tag as well. Inference should be use for the general concept, and we should find a good name for our inference (opened an issue about it), and use the tag for both in the meantime (or add a flepiMoP R inference tag that contrasts with the python inference tag and the general concept)

Reply:
okay i'll change it to just say `config`

Reply:
both config and flepiconfig are need

Reply:
okay. in your words, what is the difference between the two?

Reply:
flepiconfig affects the flepiconfig packages that writes configs (create bad configs, features to add). 
config could be named configuration and deals with the syntax and features supported in the configuration files